recipes/diligent-core/all/conanfile.py

```diff
@@ -28,9 +28,7 @@ class DiligentCoreConan(ConanFile):
         "fPIC": True,
         "with_glslang": True
     }
-    generators = "cmake_find_package", "cmake", "cmake_find_package_multi"
-    _cmake = None
-    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "CMakeToolchain", "CMakeDeps", "cmake_find_package"
```

Comment:
```suggestion
    generators = "CMakeToolchain", "CMakeDeps", "cmake_find_package"
```
Why do you need to mix cmake_find_package with CMakeDeps?